### PR TITLE
removed references to deprecated python_utils file

### PR DESCRIPTION
**Pull request by athyrson06**
## **Changes Following [#13935](https://github.com/oppia/oppia/issues/13935)**

In response to [#13935](https://github.com/oppia/oppia/issues/13935), several misleading mentions and incorrect warnings were removed from the wiki. These included:

- **Removal of `python_utils` mentions**: The previous examples using on `python_utils.OBJECT` and other utility functions has been updated. I’ve replaced them with standard Python constructs and best practices.
- **Fixing the incorrect warning about `str` function**: Some parts of the code were erroneously issuing warnings regarding the use of the `str()` function. This warning was removed as `str()` is the correct and accepted way to convert objects to strings in Python 3.

### **Key Changes:**
- **Removed references to `python_utils.OBJECT`** in class definitions, replacing them with standard Python class definitions.
- **Fixed the incorrect warning about using `str()`**: The previous message warning against using `str()` was incorrect, as `str()` is the appropriate way to convert to strings in Python 3.

These updates help modernize the wiki, making it more in line with the new Oppia standard.

### CODE REVIEW

**Comment by seanlip** [comment]
@athyrson06 Please follow the "getting started" instructions in the Oppia wiki https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up -- we can't accept your PRs until you've done that.

Once you've done that, feel free to reopen this PR if you like, and we can review it. I'm closing it for now so that it doesn't get merged accidentally. Thanks!

**Comment by athyrson06** [comment]
I have now followed the steps, and I would like to request the pull request to be reopened.

File: Coding-style-guide.md [hunk 1]
```diff
@@ -125,25 +105,6 @@ If, in some case, an external source returns or receives data with a different e
         input_pcoll | "Sort" >> Sort() | "Unique" >> Unique())
     ````
 
-
-## Prettier
```
Comment (seanlip) [inline Coding-style-guide.md]: Why is this deleted?

File: Custom-Pylint-checks.md [hunk 2]
```diff
@@ -499,11 +499,11 @@ def test_finds_hello_world_assignment(self):
     temp_file = tempfile.NamedTemporaryFile()
     filename = temp_file.name
 
-    with python_utils.open_file(filename, 'w') as tmp:
-        tmp.write("""
-        s = "Hello, world!"
-        print("Hello, world!")
-        """)
+    with open(filename, 'w') as tmp:
+            tmp.write("""
```
Comment (seanlip) [inline Custom-Pylint-checks.md]: Deindent this and the following lines by 4.

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks @athyrson06! Just two comments, PTAL.

File: Coding-style-guide.md (hunk 1 above)
Comment (athyrson06) [reply Coding-style-guide.md]: I think it is a mistake. Will fix it.

File: Custom-Pylint-checks.md (hunk 2 above)
Comment (athyrson06) [reply Custom-Pylint-checks.md]: done

**Comment by athyrson06** [comment]
I did the necessary adjustments now.

**Review by seanlip — APPROVED** [review]
Thanks @athyrson06, looks good!  Congrats on your first Oppia PR :tada:
